core/src/mindustry/ui/dialogs/SettingsMenuDialog.java

```diff
@@ -270,26 +270,26 @@ void rebuildMenu(){
 
         TextButtonStyle style = Styles.flatt;
 
-        float marg = 8f, isize = iconMed;
+        float isize = iconMed;
 
-        menu.defaults().size(300f, 60f);
-        menu.button("@settings.game", Icon.settings, style, isize, () -> visible(0)).marginLeft(marg).row();
-        menu.button("@settings.graphics", Icon.image, style, isize, () -> visible(1)).marginLeft(marg).row();
-        menu.button("@settings.sound", Icon.filters, style, isize, () -> visible(2)).marginLeft(marg).row();
-        menu.button("@settings.language", Icon.chat, style, isize, ui.language::show).marginLeft(marg).row();
+        menu.defaults().size(300f, 60f).marginLeft(8f);
```

Comment:
Okay, so I go into the settings, and you did *not* test this, because it doesn't work. The margin is messed up; it is not preserved through defaults. Reverting.

Reply:
Damn it, everything but the generation is messed up in this, how did I not notice that.

![a.jpg](https://user-images.githubusercontent.com/85090668/177068019-9e40370f-a038-4eb7-87d3-58863c94313b.jpg)

Nowadays I can't trust my changes, very suspicious.